msatutil/mair_geoviews.py

```diff
@@ -206,6 +208,7 @@ def show_map(
         title=title,
         alpha=alpha,
         active_tools=active_tools,
+        tools=tools,

```

Comment:
does this still keep the default tools (pan/box_zoom/wheel_zoom/reset) when the list is empty?

Reply:
Yes. The active_tools options will still be turned on. There seems to be some bug where the hover tool can only be specified under the `tools` argument and the not `active_tools`, which is why I added this argument.

```python
show_map(lon, lat, var_list[0], single_panel=True, tools=['hover'])
```
![image](https://github.com/user-attachments/assets/46543904-dcb6-4925-b1fa-40ee33bf55e8)


```python
show_map(lon, lat, var_list[0], single_panel=True)
```
![image](https://github.com/user-attachments/assets/4f218b8a-cc0a-417d-b678-7969b86e75d6)